coprocessor/fhevm-engine/zkproof-worker/src/verifier.rs

```diff
@@ -356,12 +356,14 @@ pub(crate) fn verify_proof(
 
     let _s = t.child_span("create_ciphertext");
 
+    let hash_domain_separator = *b"ZK-w.rct";
     let mut h = Keccak256::new();
     h.update(raw_ct);
+    h.update(hash_domain_separator);
```

Comment:
same remark about domain separator being the last, usually it's the first thing to hash